### PR TITLE
Tb205gti accelfor p

### PR DESCRIPTION
**Pull request by tb205gti**
Accell limits for Performance model S, changes to both accel limits and PI feed values.

### CODE REVIEW

File: selfdrive/manager.py [hunk 1]
```diff
@@ -536,8 +536,8 @@ def main():
   if os.getenv("PREPAREONLY") is not None:
     spinner_proc = None
   else:
-    spinner_text = "chffrplus" if params.get("Passive")=="1" else "openpilot"
-    spinner_proc = subprocess.Popen(["./spinner", "loading %s"%spinner_text],
+    spinner_text = "chffrplus" if params.get("Passive")=="1" else "Tesla P85+"
```
Comment (rafcabezas) [inline selfdrive/manager.py:539]: Shouldn't it just be "Tesla" for the else here?

Comment (tb205gti) [reply selfdrive/manager.py:539]: Woops that was not supposed to be part of the PR. :/ That is solely for myself. It just needs to be openpilot for all others.

**Comment by jeankalud** [comment]
Just saw this, will test and comment back!

**Comment by jeankalud** [comment]
Much better, still some brief brake light sometimes (too harsh decel) but much better most of the time

**Comment by rafcabezas** [comment]
@tb205gti do we need another pr to correct the unintended change to manager.py now that this pr got merged in?

**Comment by BogGyver** [comment]
Just edit the manager.py back straight into GitHub if you want. 

Bogdan Butoi
610.329.6720

**Pull request merged**